Remotion/Web/ClientScript/Scripts/ListMenu.ts

```diff
@@ -114,31 +114,25 @@ class ListMenu
       const item = document.getElementById (itemInfo.ID) as HTMLElement;
       let anchor = item.children[0] as HTMLAnchorElement;
       const icon = anchor.children[0] as HTMLImageElement;
+      anchor.removeAttribute ('href');
+      anchor.removeAttribute ('target');
+      anchor.removeAttribute ('onclick');
+      anchor.onclick = null;
```

Comment:
`removeAttribute` will also clear up the event handler so there is no need to set it to `null` afterwards.

Reply:
interesting.